tpp-rt/XsmmRunnerUtils.cpp

```diff
@@ -204,6 +204,13 @@ _mlir_ciface_xsmm_unary_invoke(const libxsmm_datatype dType, int64_t addr,
   kernel(&param);
 }
 
+extern "C" void
+_mlir_ciface_xsmm_unary_invoke_inline(const libxsmm_datatype dType,
```

Comment:
Why do we need an extra function? Can we not use `unary_invoke` and `unary_dispatch`?

Reply:
Unary invoke takes 4 arguments, whereas in this case, we only need 3. If we use the same unary_invoke as the 4 argument version, we'll have to somehow identify that the second UnrankedMemref operand is not used. Instead, I used a different function for inlined ops which has just 3 arguments. In case we want to inline some other op going further, we can reuse this function. Same dispatch is being used right now.

Reply:
What is the other unary op? tpp.identity. I think we should have only 3 arguments, and avoid duplicating code. I am fine merging as is but please add a TODO and issue there. We can fix it later.

Reply:
Up to you (I already approved this PR).

Reply:
I'll leave a todo and an issue to refactor inline functions to see if we can simplify this further and merge for now.